packages/react-components/react-alert/src/components/Alert/renderAlert.tsx

```diff
@@ -2,12 +2,14 @@ import * as React from 'react';
 import { getSlots } from '@fluentui/react-utilities';
 import type { AlertState, AlertSlots } from './Alert.types';
 
-/**
- * Render the final JSX of Alert
- */
 export const renderAlert_unstable = (state: AlertState) => {
   const { slots, slotProps } = getSlots<AlertSlots>(state);
 
-  // TODO Add additional slots in the appropriate place
-  return <slots.root {...slotProps.root} />;
+  return (
+    <slots.root {...slotProps.root}>
+      {slots.icon && <slots.icon {...slotProps.icon} />}
+      {slots.content && <slots.content {...slotProps.content} />}
```

Comment:
I think you can render the `content` slot without a check since it's not optional in your types

```suggestion
      <slots.content {...slotProps.content} />
```